tap_closeio/streams.py

```diff
@@ -165,13 +165,25 @@ def sync_activities(ctx):
     # calls that are in progress _while_ an extraction is happening, no
     # matter the replication frequency or call duration.
     offset_secs = ctx.config.get("activities_window_seconds", (60 * 60 * 24))
+    date_window = int(ctx.config.get("activities_date_window", 15))
```

Comment:
Needs to update the readme and sample config.

Reply:
Updated README file